### ISSUE TITLE
Feature Request: Map Screenshot to Joystick

### ISSUE THREAD

**Issue by HiroiSekai** [MiSTer-devel/Main_MiSTer#558]
I don't specifically use a keyboard for the MiSTer, as I have it connected to my actual computer. I'd love to be able to map the screenshot option to a joystick button so I can share them with ease. Right now, I've been tossing up between just taking a bad photo of my monitor with my phone or unpairing/repairing my computer's keyboard just to get one screenshot.

(Unless there's a way to do this already, but I couldn't find it.)

**Comment by birdybro** [comment]
https://github.com/MiSTer-devel/Main_MiSTer/issues/184 - repeat issue.

>
>Use keyboard remap function in Menu core to map any key to PrnScr.

**Comment by thehughhefner** [comment]
@birdybro Can you bind printscreen to a button on a controller? Is that what keyboard remap enables?

**Comment by thehughhefner** [comment]
Yeah, not sure why birdybro linked that closed issue ticket to this ticket...

**Comment by HiroiSekai** [comment]
It's okay, I can understand. It has a shared issue of not being able to use the PrtScn key directly as it's programmed on the MiSTer and having to find a workaround of some sort. The other ticket was okay using 2 keyboards. I unfortunately cannot (only have one keyboard which is already tied to my computer, plus it doesn't have a PrtScn button), so I'd just like to see it mappable on a joystick/controller directly. I'm kinda used to having one set to my controller when using emulators too, for immediate snaps of unexpected moments or for specific spots during speedrunning.

**Comment by birdybro** [comment]
> Yeah, not sure why birdybro linked that closed issue ticket to this ticket...

My mistake, I misunderstood. :)

Yes, this is currently not implemented.

**Comment by HiroiSekai** [comment]
No worries :)

Any chance it could be considered in the future?

**Comment by sorgelig** [comment]
May be in far future and on 99.9% cases there are no spare buttons for this function. It's not considered as something important.
Also, screenshot doesn't include filters and shadowmask, so it won't really represent what you see on your display.

**Comment by HiroiSekai** [comment]
Far future would still be okay for me personally. I skip mapping several buttons because I never use them (mouse inputs specficially). I wouldn't mind how accurate it looked compared to the display either personally, as long as it captures the moment of gameplay in any form.

**Comment by thehughhefner** [comment]
@sorgelig It would be nice to be able to define a 2 button combination when defining joystick (same as when you define OSD button). See it as an option
Thanks,
